.changeset/selfish-rabbits-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': minor
+---
+
+Fixing Jest configuration to be able to debug tests via IDE
```

Comment:
This file should be removed now, and you should revert the entire yarn.lock as well